.github/src/exec.js

```diff
@@ -16,7 +19,7 @@ export async function execRoot(command, core) {
     cwd: process.env.GITHUB_WORKSPACE,
```

Comment:
I came looking to see how we handle errors but I see there is a TODO for that. We do need to start looking at exit codes and other things for these commands.

Reply:
I think handling errors at this level is requiring too much of `execRoot`. If `execRoot` returned the entire `result` object that would include `error`, `stdout`, and `stderr`. 

The calling function would need to evaluate something like `error.code`.

Reply:
I see... `try/catch` is reasonable in this context https://github.com/Azure/azure-rest-api-specs/pull/33133/files#diff-2f4c43b6eef906041430c97b74469c65d9b3d8fcc22598593df91c90ed745e2fR70

Reply:
I think we should add an optional parameter like `throwOnError` that defaults to `true`, but can be overridden for the rare cases you don't want to throw.  We may also need to return all of {error, stdout, and stderr} instead of just stdout.

https://github.com/Azure/azure-rest-api-specs/issues/33192